packages/api/src/command/mapping/facility.ts

```diff
@@ -53,24 +54,58 @@ export async function getFacilityMappingOrFail({
   return mapping;
 }
 
-export async function getFacilityMappingsForCustomer(where: {
+export async function getFacilityMappingsByCustomer({
+  cxId,
+  source,
+}: {
   cxId: string;
   source?: string;
 }): Promise<FacilityMapping[]> {
-  const rows = await FacilityMappingModel.findAll({ where });
-  return rows.map(r => r.dataValues);
+  const mappings = await FacilityMappingModel.findAll({
+    where: { cxId, ...(source && { source }) },
+  });
+  return mappings.map(m => m.dataValues);
 }
 
-export async function deleteFacilityMapping({
+async function getFacilityMappingModelById({
   cxId,
-  externalId,
-  source,
-}: FacilityMappingLookUpParams): Promise<void> {
+  id,
+}: FacilityMappingLookupByIdParams): Promise<FacilityMappingModel | undefined> {
   const existing = await FacilityMappingModel.findOne({
-    where: { cxId, externalId, source },
+    where: { cxId, id },
   });
-  if (!existing) {
-    throw new NotFoundError("Entry not found", undefined, { cxId, externalId, source });
+  if (!existing) return undefined;
+  return existing;
+}
+
+async function getFacilityMappingModelByIdOrFail({
+  cxId,
+  id,
+}: FacilityMappingLookupByIdParams): Promise<FacilityMappingModel> {
+  const mapping = await getFacilityMappingModelById({
+    cxId,
+    id,
+  });
+  if (!mapping) {
+    throw new NotFoundError("FacilityMappingModel not found", undefined, { cxId, id });
```

Comment:
nit: when I read this message I think the file/class is not available... 😄

What do you think we reword it to `Facility mapping not found`?

Reply:
done.